docs/reference/team-processes/development-process.md

```diff
@@ -296,7 +296,7 @@ The Mozilla accounts service is part of Mozilla's [bug bounty program][moz-bug-b
 
 
 
-[bugzilla-triage-process]: https://mozilla.github.io/bug-handling/triage-bugzilla
+[bugzilla-triage-process]: https://firefox-source-docs.mozilla.org/bug-mgmt/policies/triage-bugzilla.html
```

Comment:
**Note:** I have updated this reference only if it ever gets used again — I believe it was superseded by: https://github.com/mozilla/ecosystem-platform/blob/802fc4af6373008670af255bdff1231be2cde2fd/docs/reference/team-processes/development-process.md?plain=1#L322

at some point in the past. If you feel the local definition is sufficient and the source-docs link won't be needed going forward, I can remove it completely instead of updating:

```suggestion
```